README.md

```diff
@@ -145,6 +147,22 @@ $ cat /tmp/20200710182129_ksvc_creation_time.csv
   ktest-109,ktest-1,13,13,2,0,0,0,0,15,2,0,2,15
   ktest-110,ktest-1,14,13,1,0,0,0,0,18,3,0,3,18
 ```
+
+### Clean Knative Service generated for test
+```shell script
+# Delete all ksvc with name prefix ktest in namespaces with name prefix test and index 1,2,3
+$ kperf service clean --nsPrefix test --nsRange 1,3 --svcPrefix ktest
```

Comment:
According to the latest code:
`nsPrefix` => `namespace-prefix`
`nsRange` => `namespace-range`
`svcPrefix ` => `svc-prefix`

Reply:
fixed!